frontend/mock-api/index.js

```diff
@@ -151,8 +151,14 @@ module.exports = function (app, port) {
               status: "DONE",
               numberOfResults: 5,
               resultUrls: [
-                `/api/results/results.xlsx`,
-                `/api/results/results.csv`,
+                {
+                  label: "XLSX",
+                  url: "http://localhost:8080/api/result/xlsx/mimic-iii-demo.51cd95fd-90b2-4573-aab5-11846126427b.xlsx",
```

Comment:
Where do these names + ids come from? `mimic-iii-demo.51cd95fd-90b2-4573-aab5-11846126427b`

Reply:
@fabian-bizfactory ?

Reply:
they are propably from the mimic III tutorial dataset

Reply:
Yes, they are from the tutorial dataset. The ids are generated from the backend.